### DISCUSSION TITLE
Prompt on a new line?

### DISCUSSION THREAD

**Discussion by ghost** [liquidprompt/liquidprompt#819] [Q&A] [answered]
Using the powerline theme... Is there a way to have the prompt itself where I'm typing a command always be on a new line on the left hand side? Powerline tends to push my prompt all the way to the right and more than half of my commands I write end up wrapping, and I'd rather just have the prompt itself start on a new line and have the status information on the line above.  Thanks.

**Comment by Rycieos** [comment, marked as answer]
This likely won't be the conclusive answer you were hoping for. That is because we are in the process of reworking the Powerline themes, breaking them out into their own repo: https://github.com/liquidprompt/liquidprompt-powerline. Once that is done, the version of the themes embedded in this repo will be deprecated.

So, the answers for the themes in this repo are:
* For the `powerline` theme: no, there is no supported way to do that currently.
* For the `powerline_full` theme, you can set `LP_PS1_POSTFIX="$NO_COL"$'\n'`.
* For both, you can override the `__powerline_end_terminator()` function to add a `"$NO_COL"$'\n'` like so, which will look slightly better than the above option:
```bash
 __powerline_end_terminator() {
    # Always add a end terminator, even if nothing is before it
    local first_section="true"
    __powerline_section_arrow -1
    lp_terminal_format -1
    powerline_sections+="$NO_COL"$'\n'"${section_arrow}${lp_terminal_format}${POWERLINE_SPACER}"
}
```

The answers for the new repo:
* For the `powerline` theme, same as above.
* For the `powerline_plus` theme, same as `powerline_full` above. (This is a renamed and expanded `powerline_full`.)
* Use one of the new `power2lines` or `powerpuff` themes, which both have this feature by default (and will look better than the two previous options).

**Comment by ghost** [comment]
Thank you!
